.mergify.yml

```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: automatic merge on CI success and review
```

Comment:
Actually... do we wan't fully automatic merge? Maybe let's leave it up to the author when to merge?... or at least make everybody aware that merges happen automatically... does mergify notify reviewers that "once you press the Approve button, the PR will be merged"?

Reply:
Mergify does automatic merge if conditions are fulfilled. It will notify that PR was merged automatically.

If you don't want to have it right now, let skipe it.

Reply:
Yeah, let's skip this as for now

Reply:
OK.